src/utils/format.js

```diff
@@ -46,21 +46,21 @@ export const getAssetDenomination = (
 ): ?string => {
   switch (denomination) {
     case ASSET_DENOMINATION.TICKER:
-      return token.metadata.ticker
+      return token?.metadata?.ticker
```

Comment:
token is guaranteed by flow, there is no need for `token?.`

Reply:
It was failing when I have been running it on simulator 🤔 It might be some kind of my local problem, but it looked like the metadata was non-existing when the send page was just opened for the first time, as no value have been selected in the drop-down yet. I will try to look into removing uneccessary checks later

Reply:
I will do some investigation into whether no selected value in the drop-down will cause the `token` to be empty as well, if not - will remove the checks and see why it could have been failing in my simulator. But if yes - will review the flow types to make sure that's reflected there

Reply:
@vantuz-subhuman it was happening to me as well, but only at first run.

Reply:
hmmmmmmm 🤔

Reply:
@vantuz-subhuman did you find anything out yet?